src/app/pages/about-us/components/rating/rating.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="flex mt-2 mb-5 ">
```

Comment:
Instead of adding extra DIV container, use HostBinding('class') decorator for it